Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "elasticsearch_exporter"
+version = "0.1.0"
+authors = ["Ernestas Poskus <ernestas@vinted.com>"]
+edition = "2018"
+description = """
+Vinted Elasticsearch exporter
+"""
+documentation = "https://docs.rs/elasticsearch_exporter"
+homepage = "https://github.com/vinted/elasticsearch_exporter.rs"
+license = "BSD-3-Clause"
+repository = "https://github.com/vinted/elasticsearch_exporter.rs"
```

Comment:
I'd probably change repo name to `rust-elasticsearch-exporter` or `elasticsearch-exporter-rs` to align better with other community projects.

Reply:
.rs is also common, since `elasticsearch_exporter` is already occupied by Golang fork, I will remove dot and go with dash variant `-rs`

Reply:
@ernestas-poskus nit, but I'd use consistent hyphens: `elasticsearch_exporter-rs` -> `elasticsearch-exporter-rs`.